plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/bls12_381_G1_neg/add-neg/add-neg.uplc

```diff
@@ -0,0 +1,13 @@
+-- Check that adding a random point to its negative gives the zero element.
+(program 1.0.0
```

Comment:
> Why not property tests? Here and in other places.

Pure laziness (appropriately for Haskell).  There are property tests for things like that [here](https://github.com/input-output-hk/plutus/blob/513f4e63f72cd90b02fdf19c1e97ff64b6ecb95d/plutus-core/untyped-plutus-core/test/Evaluation/Builtins/BLS12_381.hs#L80), and it was a lot easier to put those in `plutus-core`.  Ultimately we should probably make all of those (and also tests for other builtins) into conformance tests.  There's a ticket for that at PLT-1303; I guess it would make sense to do that separately.

Reply:
> T[here](https://github.com/input-output-hk/plutus/blob/513f4e63f72cd90b02fdf19c1e97ff64b6ecb95d/plutus-core/untyped-plutus-core/test/Evaluation/Builtins/BLS12_381.hs#L80) are property tests for things like that here, and it was a lot easier to put those in plutus-core.

That's a pretty good reason! Thanks.